package.json

```diff
@@ -36,7 +36,8 @@
     "example": "yarn --cwd examples/default",
     "hybrid": "yarn --cwd examples/hybrid",
     "pods": "cd examples/default &&  pod-install --quiet && cd ../hybrid && pod-install --quiet ",
-    "bootstrap": "yarn example && yarn hybrid && yarn && yarn pods"
+    "packSDK": "yarn build && yarn pack --filename InstabugSDK.tgz",
+    "bootstrap": "yarn packSDK && yarn example && yarn hybrid && yarn && yarn pods"
```

Comment:
We probably don't need `yarn packSDK` in the `bootstrap` scripts, it's only needed when we want to build the app for a build check like we do on CI.